modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/TaxonomyInfoCore_conf.pm

```diff
@@ -0,0 +1,119 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2022] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::TaxonomyInfoCore_conf;
+
+use strict;
+use warnings;
+
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::Base_conf');
+
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use Bio::EnsEMBL::Hive::Version 2.5;
+use File::Spec::Functions qw(catdir);
+
+sub default_options {
+  my ($self) = @_;
+  return {
+    %{$self->SUPER::default_options},
+
+    # Primary function of pipeline is to copy, but can be used to just delete dbs
+    username => $self->o('ENV', 'USER'),
+    ensembl_release => $self->o('ENV', 'ENS_VERSION'),
+    dumppath => "/hps/scratch/flicek/ensembl/". $self->o('username') ."/db_backs_meta_tables/". $self->o('ensembl_release'),
```

Comment:
May be scratch dir (only 20Gb) will be too small for the dumps in case we run it for all species. `Base_conf` define as well some `large_scratch_dir` option, which may be reused.